code/obj/table.dm

```diff
@@ -532,6 +535,22 @@ TYPEINFO_NEW(/obj/table/scrap)
 	auto
 		auto = 1
 
+TYPEINFO(/obj/table/mauxite)
+TYPEINFO_NEW(/obj/table/mauxite)
+	. = ..()
+	smooth_list = typecacheof(/obj/table/mauxite/auto)
+/obj/table/mauxite
+	name = "table"
+	icon = 'icons/obj/furniture/table.dmi'
+	icon_state = "0$$mauxite"
+	desc = "A table made of dense mauxite"
```

Comment:
Won't the description already get changed by the mat to add the mauxite bit? (i.e. isn't this kinda redunant and possibly false if the table gets transmuted later?)